sway/input/seat.c

```diff
@@ -1090,8 +1090,19 @@ void seat_configure_xcursor(struct sway_seat *seat) {
 bool seat_is_input_allowed(struct sway_seat *seat,
 		struct wlr_surface *surface) {
 	struct wl_client *client = wl_resource_get_client(surface->resource);
```

Comment:
The client declaration here can be moved down to where it's used. The lock code doesn't need to use it. Otherwise, LGTM

Reply:
Fixed, thanks!